tsconfig.check.json

```diff
@@ -3,14 +3,14 @@
     "allowJs": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es2021"],
+    "lib": ["ES2021"],
```

Comment:
When this was in lower case I was hitting an error importing `Intl.ListFormat` (which was added in ES2021).

Reply:
It's less than ideal that this upper-case version is correct for [`lib`](https://www.typescriptlang.org/tsconfig#lib) but [`target`](https://www.typescriptlang.org/tsconfig#target) refers to the lowercase `es2021` ! :face_exhaling: